Sources/SwiftKuery/Table.swift

```diff
@@ -139,41 +140,6 @@ open class Table: Buildable {
         return result
     }
     
-    /// Function that returns the SQL CREATE TABLE query as a String for this TABLE.
-    /// - Returns: A String representation of the table create statement.
-    /// - Throws: QueryError.syntaxError if statement build fails.
-    public func description(connection: Connection) throws -> String {
```

Comment:
Was there a reason you moved the describe to the bottom of the class? Just thinking in terms of Jazzy docs since descriptions kind of fits within Initializer?